wavefront/server/modules/voice_agents_module/voice_agents_module/services/exotel_service.py

```
@@ -0,0 +1,112 @@
+
+        if not self.call_processing_base_url:
+            raise ValueError(
+                'call_processing_base_url is required in voice_agents config'
+            )
+
+    async def initiate_call(
+        self,
+        to_number: str,
+        from_number: str,
+        voice_agent_id: str,
+        welcome_message_audio_url: str,
```

Ruff (0.14.14)

[warning] Unused method argument: `welcome_message_audio_url`

(ARG002)